src/index.js

```diff
@@ -35,6 +40,11 @@ if (
   }
 }
 
+// Save the current package version
+if (packageVersion !== lastPackageVersion) {
+  setPackageVersion(packageVersion)
+}
```

Comment:
@sohkai Do we have a reason to not do that?

Reply:
Nope, I think setting this every time is fine :+1:.